mods/ra/rules/defaults.yaml

```diff
@@ -249,8 +249,14 @@
 	Selectable:
 		Bounds: 24, 24
 	Targetable:
-		TargetTypes: Ground, Repair, Vehicle
 		RequiresCondition: !parachute
+		TargetTypes: Ground, Vehicle
+	Targetable@REPAIR:
+		RequiresCondition: !parachute && damaged
+		TargetTypes: Repair
+	GrantConditionOnDamageState@DAMAGED:
+		Condition: damaged
+		ValidDamageStates: Light, Medium, Heavy, Critical
```

Comment:
IMO it makes the rules much easier to read if the fields that determine whether a trait is active (`RequiresCondition`, `RequiresPrerequisites`, `Faction` – and in this case `ValidDamageStates`) go at the top, before fields that define what the trait does when it is active (in this case grant `Condition`).

I don't think this needs to block merging here, but it is something I would like to start applying to our default mods.  We should consider adding a section to our coding standards with this and other conventions (e.g. for the trait `@SUFFIX`es and condition names) if we can agree on what the standards should be.